src/main/sources/BigBedDataSource.js

```diff
@@ -22,10 +22,28 @@ export type Gene = {
   name: string;  // human-readable name, e.g. "TP53"
 }
 
+export type Feature = {
+  id: string;
+  featureType: string;
+  contig: string;
+  start: number;
+  stop: number;
+  score: number;
+}
+
+// Flow type for export.
+export type FeatureDataSource = {
+  rangeChanged: (newRange: GenomeRange) => void;
+  getFeaturesInRange: (range: ContigInterval<string>) => Feature[];
+  on: (event: string, handler: Function) => void;
+  off: (event: string) => void;
+  trigger: (event: string, ...args:any) => void;
+}
+
 // Flow type for export.
 export type BigBedSource = {
   rangeChanged: (newRange: GenomeRange) => void;
-  getGenesInRange: (range: ContigInterval<string>) => Gene[];
+  getFeaturesInRange: (range: ContigInterval<string>) => Gene[];
```

Comment:
thanks for renaming this — `feature` would better speak to the bioinformaticians in this context.